### PR TITLE
Fixing create view for amendments (fixing #3004)

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/motions/views.py [hunk 1]
```diff
@@ -87,12 +87,25 @@ def create(self, request, *args, **kwargs):
...
+                    whitelist.extend([
+                        'parent_id',
+                        'category_id',      # This will be set to the matching
+                        'motion_block_id',  # values from parent_motion.
+                    ])
+                    request.data['category_id'] = parent_motion.category.id
+                    request.data['motion_block_id'] = parent_motion.motion_block.id
             for key in request.data.keys():
                 if key not in whitelist:
                     # Non-staff users are allowed to send only some data.
```
Comment (FinnStutzenstein) [inline openslides/motions/views.py]: @emanuelschuetze can you put here a `print("{} not in whitelist".format(key))`? This may give us the wrong key.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Please retest.

**Comment by emanuelschuetze** [comment]
> Please add CHANGELOG notice.

Sorry, but bugfixing is not a entry in Changelog... we already have a generic entry "Reimplemented amendments"... So won't fix. I'll merge it.

**Comment by normanjaeckel** [comment]
Ok. (I thought about this special behavior when a new motion is created (that it is set to the same category and block)).

**Pull request merged**